### ISSUE TITLE
Funds availability policy disclosure not generated

### ISSUE THREAD

**Issue by Cllyde** [Cllyde/iDeposit#1]
Creating documents for DDA account 4242 in my test environment is not producing a _Funds Availability Policy Disclosure_ document.

**Comment by Cllyde** [comment]
Looking at the DataHolder.cs file, in the LoadInsiteObjects() method, line 99. It calls 

```
InsiteConverter.setData<CsiclsdescObjDT, CsiclsdescObj>(InsiteDataTable.CSiCLSDescDT, ref CSiCLSDesc)
```

Which sets the DataHolder's `List<CsiclsdescObj> CSiCLSDesc` (base class property) to the value retrieved from the InsiteDT object that is passed into the constructor of DataHandler.

I'm guessing there is somewhere in Insite that provides the `InsiteDataTable.CSiCLSDescDT`. It sounds like a table in Insite.

tl;dr I need to find out where the list or table is being populated from before it is passed to the DataHandler constructor.
